dbflow/src/main/java/com/xmartlabs/bigbang/dbflow/controller/DbFlowController.java

```diff
@@ -56,7 +56,7 @@
                                                  @NonNull SQLOperator... conditionsToDelete) {
     return Single
         .fromCallable(() -> {
-          FlowManager.getDatabase(getAppDataBaseClass())
+          FlowManager.getDatabase(modelClass)
```

Comment:
nice!, but if the `getAppDataBaseClass()` is not needed, then this class shouldn't be abstract